src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -204,14 +204,12 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
                 from . import nircam as inst
             elif meta.inst == 'nirspec':
                 from . import nirspec as inst
-                log.writelog('WARNING: Are you using real JWST data? If so, '
-                             'you should edit the flag_bg() function in '
-                             'nirspec.py and look at Issue #193 on Github!')
             elif meta.inst == 'niriss':
                 raise ValueError('NIRISS observations are currently '
                                  'unsupported!')
             elif meta.inst == 'wfc3':
                 from . import wfc3 as inst
+                meta.bg_dir = 'CxC'
```

Comment:
Why only set this default here?

Reply:
The WFC3 test fails if I don't set it here.  The remaining instruments can all get set with a single command on Line 309.